tools/inkscape_extension/visicut_export.py

```diff
@@ -141,12 +141,19 @@ def is_exe(fpath):
 def inkscape_version():
     """Return Inkscape version number as float, e.g. version "0.92.4" --> return: float 0.92"""
     version = subprocess.check_output([INKSCAPEBIN, "--version"],  stderr=DEVNULL).decode('ASCII', 'ignore')
+    if not version.startswith("Inkscape "):
```

Comment:
No need for this check, you can run this line-wise "grep" every time.

Reply:
I am just defensive here. By testing the assert condition already here, I am sure, that whatever my code breaks, it will only break in cases that would have failed before anyway.

Reply:
I would prefer a simpler version that always does the same. The currently proposed code is a little confusing because in one branch, version "leaks" from the for loop, while in the other branch, it is the original version variable.

Maybe something like
```
version_raw = subprocess....(...)
# get first line of stdout that starts with "Inkscape"
lines = version_raw.splitlines()
version = [line for line in lines if line.startsWith("Inkscape ")]
assert length(version) == 1, "inkscape --version did not return version number"
```

Reply:
Done that. Please re-review.